resolver/resolver.go

```diff
@@ -203,6 +203,15 @@ type State struct {
 // gRPC to add new methods to this interface.
 type ClientConn interface {
 	// UpdateState updates the state of the ClientConn appropriately.
+	//
+	// If an error is returned, the resolver should try to resolve the
+	// target again. Ideally the resolver uses a backoff timer to prevent
+	// overloading the server with requests.
+	// If a resolver is certain that reresolving does not change the result,
```

Comment:
"does not change" -> "will not change" or "would not change"

Reply:
changed to "would not change"